wdl-lint/tests/lints/disallowed-input-name/source.errors

```diff
@@ -30,3 +30,11 @@ note[DisallowedInputName]: declaration identifier starts with 'in'
    │
    = fix: rename the identifier to not start with 'in'
 
+warning[DisallowedDeclarationName]: declaration identifier 'int' ends with type name 'Int'
+   ┌─ tests/lints/disallowed-input-name/source.wdl:24:13
+   │
+24 │         Int int = 1  # This is OK
```

Comment:
I think we just delete this line from `source.wdl` to prevent this diagnostic. It's not a good example anymore